website/content/docs/k8s/installation/vault/wan-federation.mdx

```diff
@@ -0,0 +1,678 @@
+To use Vault as the Service Mesh Certificate Provider on Kubernetes, please follow the process guidelines in the [Data Integration](/docs/k8s/installation/vault/data-integration) section:
+
+- One time setup in Vault
+  1. Store the secrets in Vault.
+  1. Create a Vault policy that authorizes the desired level of access to the secrets.
+- Setup per Consul datacenter
+  1. Create Vault Kubernetes auth roles that link the policy to each Consul on Kubernetes service account that requires access.
+  1. Configure the Vault Kubernetes auth role in the Consul on Kubernetes helm chart.
+
+### One time setup in Vault
+1. Store the ACL Replication Token, Gossip Encryption Key, and Root CA certificate secrets in Vault.
+
+    ```shell-session
+    $ vault kv put consul/secret/gossip key="$(consul keygen)"
+    ```
+
+    ```shell-session
+    $ vault kv put consul/secret/replication token="$(uuidgen | tr '[:upper:]' '[:lower:]')"
+    ```  
+    ```shell-session
+    $ vault write pki/root/generate/internal common_name="Consul CA" ttl=87600h
+    ``` 
+
+1. Create Vault policies that authorize the desired level of access to the secrets.
+
+    ```shell-session
+    $ vault policy write gossip - <<EOF
+    path "consul/data/secret/gossip" {
+      capabilities = ["read"]
+    }
+    EOF
+    ```
+
+    ```shell-session
+    $ vault policy write replication-token - <<EOF
+    path "consul/data/secret/replication" {
+      capabilities = ["read"]
+    }
+    EOF
+    ```
+
+    ```shell-session
+    $ vault policy write ca-policy - <<EOF
+    path "pki/cert/ca" {
+      capabilities = ["read"]
+    }
+    EOF
+    ```
+
+### Setup per Consul datacenter
+#### Primary Datacenter (dc1)
+
+1. Change your Kubernetes current context to target the primary datacenter (dc1):
+    ```shell-session
+    $ kubectl config use-context <context for dc1>
+    ```
+
+1. Create Server TLS and Service Mesh Cert Policies
+
+    ```shell-session
+    $ vault policy write consul-cert-dc1 - <<EOF
+    path "pki/issue/consul-cert-dc1"
+    {
+      capabilities = ["create","update"]
+    }
+    EOF
+    ```
+
+    ```shell-session
+    $ vault policy write connect-ca-dc1 - <<EOF
+    path "/sys/mounts" {
+      capabilities = [ "read" ]
+    }
+    path "/sys/mounts/connect_root" {
+      capabilities = [ "create", "read", "update", "delete", "list" ]
+    }
+    path "/sys/mounts/dc1/connect_inter" {
+      capabilities = [ "create", "read", "update", "delete", "list" ]
+    }
+    path "/connect_root/*" {
+      capabilities = [ "create", "read", "update", "delete", "list" ]
+    }
+    path "/dc1/connect_inter/*" {
+      capabilities = [ "create", "read", "update", "delete", "list" ]
+    }
+    EOF
+    ```
+
+1. Create Vault Kubernetes auth roles that link the policy to each Consul on Kubernetes service account that requires access.  For each auth method in Vault, you will need roles for Consul server, Consul client, the `server-acl-init` job, and Consul server CA.
```

Comment:
```suggestion
1. Create Vault Kubernetes auth roles that link the policy to each Consul on Kubernetes service account that requires access.  For each auth method in Vault, you will need roles for the Consul server, Consul client, the `server-acl-init` job, and Consul server CA.
```

We should also make this section into a list, as that is what was done for the other section down the document.